playbooks/k8s-cluster.yml

```diff
@@ -31,6 +31,8 @@
 
 # Install Kubernetes
 # for configuration, see: config/group_vars/k8s-cluster.yml
+## 
+## PHogan
```

Comment:
Not clear why this comment is here, can we remove it?

Reply:
Its not needed